composer.json

```diff
@@ -71,7 +71,7 @@
     "require": {
         "php":                          ">=5.3.3",
         "symfony/framework-bundle":     "~2.3",
-        "symfony/security-bundle":      "~2.1",
+        "symfony/security-bundle":      "~2.6",
```

Comment:
@scaytrase this is precisely the issue of this patch: it should be written in a different way to keep compatibility with older versions too (it is totally possible to support both the older API and the new one in the same codebase instead of forcing to maintain 2 versions of the bundle, which is a huge pain for maintainers and does not play well with semantic versionning of the bundle)